Core.lua

```diff
@@ -193,6 +193,10 @@ function WarpDeplete:Show()
   self.frames.root:Show()
   self:UpdateLayout()
 
+  if IsAddOnLoaded('SylingTracker') then
+    return
```

Comment:
This should not return right away - we might add different things below this point that should not be disabled by SylingTracker being loaded. So, just surrounding hide function below with `if SylingTracker ~= nil` would be better here, I think.

Be careful that you include the timer call in the if block. If you want, you can also pull out the call + the timer call into a separate `WarpDeplete:HideBlizzardObjectiveTracker` function, that would probably be a bit cleaner.